spec/codelog/command/bump_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+
+describe Codelog::Command::Bump do
+  describe '#run' do
+    before do
+      allow(Dir).to receive_message_chain(:glob, :map).and_return(['0.0.0','1.2.3'])
```

Comment:
@Guilherme2112 stubbing the `.map` method will not allow to test the logic inside the iteration. In this specific case this logic is quite important as it defines what is and what is not considered a version, based on the file names.
I'd recommend stubbing the `glob` method to return some filenames, including some that should not be picked, like `fake_file.txt`, what do you think?